rooms.js

```diff
@@ -261,6 +261,22 @@ class Room {
 			}
 		}
 	}
+
+	get metadata() {
+		let settings = {
+			// settings
+			privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'),
+			modchat: this.modchat,
+			modjoin: this.modjoin,
+
+			// games
```

Comment:
I agree; and plus, bots /could/ use /roomsettings

I mean, I know it's not ideal and they would have to do some parsing, but I'm just saying that all this information with games /is/ already accessible to them.